components/SideMenu.js

```diff
@@ -12,11 +12,14 @@ import style from '../public/style';
 export default class SideMenu extends Component {
 
 	state = {};
+	 componentDidMount =  async () => {
+		var unsubscribe =	await store.collection('users').doc(auth.currentUser.email).onSnapshot(user => {
```

Comment:
var?

Reply:
we're not sure if its actually working cause my linter was saying that it never got used. we did var because of the global scope

Reply:
Did that fix it?

Reply:
we're not sure how to actually check if it unsubscribes

Reply:
Wouldn't it be that it stops giving us a warning? I thought that the way that we knew we had a problem was because it gave us a warning about unsubscribing. Is that right?

Reply:
we never actually had a memory leak warning tho. we just unsubed because we thought its a good idea

Reply:
Well then I think it's fine for now.. If it's not even giving us a warning, probably not a big deal even if it doesn't work. It can be something to look into later